src/modules/35-user-profile/navigation/UserNav/__tests__/UserNav.test.tsx

```diff
@@ -39,4 +40,46 @@ describe('User Profile Page', () => {
       )
     ).toBeTruthy()
   })
+
+  test('Logout with no return url', async () => {
+    mockLogout = jest.fn().mockReturnValue({ logoutUrl: undefined })
+    const { getByTestId } = render(
+      <TestWrapper path="/account/:accountId" pathParams={{ accountId: 'testAcc' }}>
+        <UserNav />
+      </TestWrapper>
+    )
+    const logoutBtn = getByTestId('signout-link')
+    act(() => {
+      fireEvent.click(logoutBtn)
+    })
```

Comment:
Instead of using `act` and `fireEvent`, you can use `userEvent` to do it in one.
```suggestion
    userEvent.click(logoutBtn)
```